usr/src/uts/common/fs/zfs/zio.c

```diff
@@ -442,18 +444,26 @@ zio_remove_child(zio_t *pio, zio_t *cio, zio_link_t *zl)
 }
 
 static boolean_t
-zio_wait_for_children(zio_t *zio, enum zio_child child, enum zio_wait_type wait)
+zio_wait_for_children(zio_t *zio, enum zio_child children,
```

Comment:
Some compilers will complain about `children` parameter, because the values passed through it are not of `enum zio_child` type.  For example, `ZIO_CHILD_VDEV | ZIO_CHILD_GANG` equals three which is not a valid value for that enumeration.